src/metabase/api/search.clj

```diff
@@ -165,13 +170,15 @@
 ;; Databases can't be archived
 (defmethod archived-where-clause "database"
   [_model archived?]
-  [:= 1 (if archived? 2 1)])
+  (if-not archived?
+    true-clause
+    false-clause))
```

Comment:
apologies for Yoda logic, but I thought "if it's not archived, then show it (otherwise don't)" read better than "if it's archived, don't show it (otherwise, do)"

Reply:
like, it just seemed a little funky to have true-clause in the _else_ and false-clause in the _then_